kafka-connect-fitbit-source/src/main/java/org/radarbase/connect/rest/fitbit/converter/FitbitIntradayCaloriesAvroConverter.java

```diff
@@ -45,6 +45,7 @@ public FitbitIntradayCaloriesAvroConverter(AvroData avroData) {
     ZonedDateTime startDate = request.getDateRange().end();
 
     return iterableToStream(dataset)
+        .filter(activity -> activity.get("value") != null && activity.get("value").asDouble() != 0)
```

Comment:
are calories ever 0? AFAIR, there is a default amount of calories that one burns at rest and those are sent as default by fitbit if no data.

Reply:
Hmm... yes after further inspection indeed that's the case. It's not that useful information, but at least it's internally consistent...